spec/models/user_spec.rb

```diff
@@ -418,8 +418,6 @@
 
     include_examples("seeding users with", [])
 
-    include_examples("seeding users with", [MiqGroup])
-
```

Comment:
@lpichler do you have any thoughts here, as this is very recent to you? The TLDR here is that I think MiqGroups cannot be seeded without MiqUserRoles and we're recently moving to actually raising errors in seeding (previously ignored). So, Users seeding with _only_ MiqGroups present shouldn't exist (the example below including MiqUserRoles should still be good). Does that make sense to you? (Sanity check for me)